### DISCUSSION TITLE
Defaulting or disabling the formatter

### DISCUSSION THREAD

**Discussion by saltysteev** [Tetrakern/fictioneer#42] [Q&A] [answered]
First and foremost - thank you for showing some love to the short story and fiction side of WP!

I'd like to set a default format font type and size for the reader, or at the very least be able to disable it and just use the Customizer options. Is this possible? Thanks again!

**Comment by Tetrakern** [comment, marked as answer]
You can change the default chapter font with the Customizer -- this will just not override already stored local settings. But anyone opening a chapter for the first time will get it as default. Probably why you did not notice, but it wouldn't do to forcefully override a reader's settings. They typically take offense to that.

As for the other settings, you'll need a child theme and this snippet: https://github.com/Tetrakern/fictioneer/blob/main/CUSTOMIZE.md#change-chapter-formatting-defaults

Disabling the formatter is not recommended. It's as much for accessibility as for preference. Alas, you could just hide the whole thing with CSS, though, or individual parts. Would not really disable it, but close enough.

**Reply by Tetrakern** [reply]
You can reset your local reader settings in the modal. Should your selected font not be the default, let me know.

**Reply by saltysteev** [reply]
Thanks for the clarification, I wasn't sure if the Customizer was default for everyone or not. And I agree, I didn't want to disable it entirely, just wanted to set defaults. Appreciate it
